frontend/public/style/_common.scss

```diff
@@ -103,6 +103,10 @@
   margin: 0 0 20px 0;
 }
 
+.co-section-title__resource-icon {
+  vertical-align: 3px;
```

Comment:
@rhamilto This feels hacky, but I couldn't get the icon vertically aligned with the heading otherwise. Let me know if there is a better way.